sympy/integrals/rubi/tests/test_utility_function.py

```diff
@@ -456,19 +568,18 @@ def test_FractionalPowerFreeQ():
     assert FractionalPowerFreeQ(x)
 
 def test_Exponent():
-    assert Exponent(x**2 + x + 1 + 5, x, Min) == 0
```

Comment:
Why were these tests changed? Especially if they were passing before they look like they are testing something different now.

Reply:
`Exponent` used to have a third optional argument which is a function that is applied to the result. This is in practice only used in `utility_function.py` for converting to a list. LGTM complained about the function sometimes returning a scalar argument, sometimes an iterable. I decided to split `Exponent` into `Exponent` and `ExponentList`. (Only `Exponent` is used in the generated code as far as I can tell, `ExponentList` is exclusively used in `utility_function`.)

Hence, testing that `Exponent` can accept a `Min` argument seemed like not really required...